### DISCUSSION TITLE
Security for temporary hosting question

### DISCUSSION THREAD

**Discussion by ghost** [yandeu/five-server#98] [Help / Questions]
Is it safe to temporarily use Five Server to host a website on the public internet for testing purposes? Are there any concerns with the security of the host PC?

**Comment by yandeu** [comment]
You mean forwarding in port of your router to your Home PC to the five server port?

If you don't expose private files such as credentials it should be fine for testing purposes.

Just make sure you only expose the pc you want and the right port. And reset the router's post forwarding options afterwards.
